casper/src/test/scala/coop/rchain/casper/util/comm/TransportLayerTestImpl.scala

```diff
@@ -0,0 +1,50 @@
+package coop.rchain.casper.util.comm
```

Comment:
We have a stub like that in `comm`, maybe worth reusing existing one (extending it if needed)? Does not have to be in this PR but don't forget about it. The smaller number of stubs the better.

Reply:
I looked at using it, but it seemed to be specialized for testing `roundTrip` as opposed to `send` and `receive`, which Casper uses. I didn't see an easy way to extend it for my purposes, which is why I wrote my own. I did use the `comm` stubs for all the other effects though.